### ISSUE TITLE
Admin migration roadmap - COMPANIES

### ISSUE THREAD

**Issue by az** [ecommerceberlin/ecommerceberlin.com#40]
# Companies

### Views

base view: list of companies sorted by brand NAME that are PRESENT in the current event scope (have paid items)

details/columns

* name
* assigned admin/account manager
* featured on website
* language
* present on current edition (has a paid purchase)

filtering base view

* present / not present at the current event scope
* account manager
* featured on website
* ?

mass actions
* assign account manager
* featured / not featured

quick lookup

* crucial company profile fields 
* comments
 
details

* same as quick lookup + editing company profile fields
* company purchases
* company comments + adding new

**Comment by az** [comment]
**Working view**

![EventJuicer 2021-08-04 14-40-38](https://user-images.githubusercontent.com/2198/128182294-ef4622ec-46ec-40d9-a4c1-dc903cb6f184.jpg)

**Comment by az** [comment]
do you need some financial data @peterecommberlin ? like total sales (all events) or avarage sales per event participated?

**Comment by az** [comment]
link to exhibitor's account

**Comment by az** [comment]
list of representatives assigned (current and pass events)

**Comment by az** [comment]
some MACRO to validate company against: 
* profile completeness (and/or freshness)
* reps added?
* upgrades?

**Comment by peterecommberlin** [comment]
> do you need some financial data @peterecommberlin ? like total sales (all events) or avarage sales (total per events participated)?

Yes, definitely. I'm preparing the concept of the data which might be useful in the admin panel.

**Comment by jprzybylski4** [comment]
tworzenie faktur na podstawie zamówień

**Comment by jprzybylski4** [comment]
dodanie pól paid/outstanding

**Comment by jprzybylski4** [comment]
w przypadku rozkładania płatności na raty możliwość automatycznego ustawienia terminu płatności drugiej raty. Optymalnie gdyby po przekroczeniu terminu w panelu admina było widać, że coś jest już przeterminowane.

**Comment by jprzybylski4** [comment]
w przypadku rozkładania płatności dodanie tagu “partial payment”

**Comment by jprzybylski4** [comment]
możliwość sprawdzania komentarzy z poprzednich edycji (szczególnie chodzi mi o to, że często ktoś otrzymuje rabat i nie wiem po czasie dlaczego)

**Comment by jprzybylski4** [comment]
statystyki dynamiki sprzedaży (nienawidzę tego liczenia), miesięczna wartość sprzedaży cash & ilość stoisk, średnia cena stoiska w danej puli, faktyczna liczba sprzedanych stoisk w ramach puli

**Comment by jprzybylski4** [comment]
opcja podpinania “rezerwowego” kontaktu pod konto wystawcy

**Comment by jprzybylski4** [comment]
podgląd aktywnych komentarzy bez wchodzenia w każdą firmę z osobna, może opcja najechania i wyskakuje pop-up

**Comment by jprzybylski4** [comment]
opcja wyłączania i włączania logotypu na stronie z poziomu admina

**Comment by jprzybylski4** [comment]
opcja dodawania ról pojedynczym sales’om np. klikam “xyz” i firma listowana jest automatycznie w sekcji featured exhibitors na stronie

**Comment by jprzybylski4** [comment]
filtrowanie firm na liście np. pokaż wszystkie nieopłacone

**Comment by jprzybylski4** [comment]
możliwość sprawdzenia historii zamówień danej firmy (items & cash, w której puli zakupili też byłoby miłą informacją)

**Comment by jprzybylski4** [comment]
Ability to track progress as in this file (step by step)
https://docs.google.com/spreadsheets/d/1OzWHNDjiwnOH3BjX1z6a-dNV5Ebjdanjo2AccdiopLM/edit#gid=956298892

**Comment by jprzybylski4** [comment]
Integration with/view of expo participation wizard
